scripts/unzip_backup.py

```diff
@@ -75,7 +82,7 @@ def push_csv_to_db(extracted_csv_path):
                         interval_start = row[2].replace(" UTC", "")
                         interval_end = row[3].replace(" UTC", "")
 
-                        if table_name_local == "0":
+                        if table_name_local == 0:
```

Comment:
nit: longer term we may want to refactor this function to avoid introducing additional complexity and keep the function's purpose tightly scoped.